### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lolothepro**
Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ and other issues with some mods.
 
 ### Manual installation (recommended)
 
-You will need Fabric Loader 0.10.x or newer installed in your game in order to load Sodium. If you haven't installed
+You will need Fabric Loader 0.12.2 or newer installed in your game in order to load Sodium. If you haven't installed
```
Comment (lusqua) [inline README.md]: Why 0.12.2? 
Do not work in old versions?

Comment (rfl890) [reply README.md]: 0.10.x is for older versions of the game. 1.12.2 is the one for 1.18.1

**Comment by Lolothepro** [comment]
https://github.com/CaffeineMC/sodium-fabric/blob/6390562feee9537461a7822b43d2f87691378e55/src/main/resources/fabric.mod.json#L33

**Comment by f478ccf2** [comment]
Why

**Pull request closed without merging**